lib/main.dart

```diff
@@ -1,6 +1,38 @@
 import 'package:flutter/material.dart';
+import 'dart:convert';
+import 'dart:async';
+import 'package:http/http.dart' as http;
+import 'package:flutter/foundation.dart';
+import 'catData.dart';
 
 void main() {
-  runApp(MyApp());
+  runApp(MainPage());
 }
 
+Future<List<CatData>> fetchCats(http.Client client) async {
```

Comment:
move this to separate files and packages.

Reply:
fixed